src/styles/index.css

```diff
@@ -47,10 +47,26 @@
 
     --p-color-live: #6272FF;
 
-    --p-color-sentiment-positive: #22C55E;
-    --p-color-sentiment-neutral: #70707B;
-    --p-color-sentiment-warning: #F5A50A;
-    --p-color-sentiment-negative: #D92D21;
+    --p-color-sentiment-positive-hue: 142deg;
+    --p-color-sentiment-positive-saturation: 71%;
+    --p-color-sentiment-positive-lightness: 45%;
+    --p-color-sentiment-positive-hsl-values: var(--p-color-sentiment-positive-hue) var(--p-color-sentiment-positive-saturation) var(--p-color-sentiment-positive-lightness);
+    --p-color-sentiment-positive: hsl(var(--p-color-sentiment-positive-hsl-values));
```

Comment:
I kept these variables which include the color space function b/c other css variables rely on them